packages/fxa-settings/src/models/Account.ts

```diff
@@ -7,10 +7,10 @@ import {
 } from '@apollo/client';
 
 export interface Location {
-  city: string;
-  country: string;
-  state: string;
-  stateCode: string;
+  city: string | undefined;
+  country: string | undefined;
+  state: string | undefined;
+  stateCode: string | undefined;
```

Comment:
I believe these could be:

```
  city?: string;
  country?: string;
  state?: string;
  stateCode?: string;
```

Reply:
I've tried `city?`, using `Partial<Location>` and this ^. None seem to fix my type error when running tests. Any suggestions?

```
Summary of all failing tests
 FAIL  src/components/ConnectedServices/index.test.tsx
  ● Connected Services › renders "fresh load" <ConnectedServices/> with correct content

    Failed: MissingFieldError {
      "clientOnly": false,
      "message": "Can't find field 'city' on object {}",
      "path": Array [
        "account",
        "attachedClients",
        0,
        "location",
        "city",
      ],
      "query": Object {
        "definitions": Array [
          [Object],
        ],
        "kind": "Document",
        "loc": [Object],
      },
      "variables": Object {},
    }

      10 | 
      11 | describe('Connected Services', () => {
    > 12 |   it('renders "fresh load" <ConnectedServices/> with correct content', async () => {
         |   ^
      13 |     renderWithRouter(
      14 |       <MockedCache account={{ attachedClients: MOCK_SERVICES }}>
      15 |         <ConnectedServices />

      at Env.it (../../node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:91:24)
      at Suite.<anonymous> (src/components/ConnectedServices/index.test.tsx:12:3)
      at Object.<anonymous> (src/components/ConnectedServices/index.test.tsx:11:1)
```

Reply:
Thanks for helping me fix this @dannycoates !